src/airflow/dags/gwas_catalog_harmonisation.py

```diff
@@ -14,7 +14,9 @@
 CLUSTER_NAME = "otg-gwascatalog-harmonisation"
 AUTOSCALING = "gwascatalog-harmonisation"
 
-SUMMARY_STATS_BUCKET_NAME = "open-targets-gwas-summary-stats"
+SUMMARY_STATS_BUCKET_NAME = "gwas_catalog_data"
```

Comment:
We need to try this, not 100% sure if all these changes are correct.

Reply:
changes make sense. It's just a matter of running it